packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -249,10 +245,34 @@ export const Product = ({
       <h3 className="billing-title">
         <span>Billing Information</span>
       </h3>
+
+      <ErrorMessage isVisible={!!createTokenError.error}>
+        {createTokenError.error && (
```

Comment:
There is a little bit of redundancy around the checks with this approach or we get complaints from React, but it's so much better than it was. 👌